app/util/AWS.scala

```diff
@@ -24,6 +25,15 @@ class AWSConfig @Inject() (config: Configuration) {
 
   lazy val credProvider = new AWSCredentialsProviderChain(credProviders.flatten: _*)
 
+  lazy val sessionId: String = "session" + Math.random()
+
+  lazy val kinesisArn = config.getString("aws.kinesis.streamArn").getOrElse("")
+
+  lazy val atomsCredProvider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("composer"),
```

Comment:
Does this mean the CODE/PROD instances will have a set of credentials in the config files?